src/app/zap-templates/zcl/data-model/chip/software-diagnostics-cluster.xml

```diff
@@ -35,7 +41,11 @@ limitations under the License.
     <attribute side="server" code="0x02" define="CURRENT_HEAP_USED" type="INT64U" min="0x0000000000000000" max="0xFFFFFFFFFFFFFFFF" writable="false" default="0x0000000000000000" optional="true">CurrentHeapUsed</attribute>
     <attribute side="server" code="0x03" define="CURRENT_HEAP_HIGH_WATERMARK" type="INT64U" min="0x0000000000000000" max="0xFFFFFFFFFFFFFFFF" writable="false" default="0x0000000000000000" optional="false">CurrentHeapHighWatermark</attribute>
     <command source="client" code="0x00" name="ResetWatermarks" optional="false" cli="chip software_diagnostics resetwatermarks">
-      <description>Reception of this command SHALL reset the values: The StackFreeMinimum field of the ThreadMetrics attribute, CurrentHeapHighWaterMark attribute</description>
+      <description>Reception of this command SHALL reset the values: The StackFreeMinimum field of the ThreadMetrics attribute, CurrentHeapHighWaterMark attribute.</description>
     </command>
+    <event side="server" code="0x00" name="SoftwareFaultEvent" priority="info" optional="true">
```

Comment:
The spec does not define the priority of this event (and in general seems to have the table defining this event in the wrong format).  Is there a spec issue tracking that?

Reply:
https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/4642